puppet/zulip/files/hooks/pre-deploy.d/zulip_notify.hook

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+
+source "$(dirname "$0")/../common/zulip_notify.sh"
+zulip_send "Starting deploy from $from to $to"
```

Comment:
These probably should include the host being deployed?

Reply:
Sure, though the `environment` is generally specific enough.